.tekton/pipeline-service-test.yaml

```diff
@@ -2,10 +2,10 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  name: pipeline-service-ckcp-test
+  name: pipeline-service-test
```

Comment:
Typos in the commit title: `change ci to aligh with the refacored pipeline-service repo` -> `change ci to align with the refactored pipeline-service repo`

Reply:
It does not matter since all changes will be squashed.